scripts/release.sh

```diff
@@ -26,8 +26,7 @@ done
 # Rebuild to stamp the release podspec
 bazel build --config=release //:PlayerUI_Podspec //:PlayerUI_Pod
 
-# VScode extension publishing
-bazel run --config=release //language/vscode-player-syntax:vscode-plugin.publish
```

Comment:
should this VScode extension publishing be removed?

Reply:
yeah this target is gone now, so it was just throwing errors

functionality was moved to the [tools repo](https://github.com/player-ui/tools/tree/main/language)